### DISCUSSION TITLE
code sample how to: mouse move, mouse click, mouse right-click, mouse click-hold-move, keyboard type

### DISCUSSION THREAD

**Discussion by codemodify** [linuxdeepin/developer-center#8950] [Development Questions | 开发答疑]
basically I'm trying to do what "robotgo" does but without the CGO, and I want to 
- mouse move
- mouse click
- mouse right-click
- mouse click-hold-move (move window, text select)
- generate keyboard events same as typing

referring to this project https://github.com/linuxdeepin/go-x11-client

**Comment by BLumia** [comment]
Sorry, which project you're referring to?

**Reply by codemodify** [reply]
sorry, updated the questions, this one https://github.com/linuxdeepin/go-x11-client
